### PR TITLE
Fixes #241 - Remove unnecessary console logs from ModifySkill component

### LINKED ISSUE #241: Remove unnecessary console logs from ModifySkill component
**Actual Behaviour**

There are unnecessary console logs which need to be removed based on comments in https://github.com/fossasia/susi_skill_cms/pull/220#issuecomment-323522015 

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by rishiraj824**
Fixes issue #241 

Changes: As per mentioned in https://github.com/fossasia/susi_skill_cms/pull/220#issuecomment-323522015
 - Removed unnecessary console logs
-  Removed Inline stylings

Link to deployment for testing:
http://cms-testing.surge.sh

@sudheesh001 @isuruAb @madhavrathi @uday96 @dynamitechetan @saurabhjn76 Please review.

### CODE REVIEW

File: src/components/SkillEditor/SkillEditor.js [hunk 1]
```diff
@@ -510,6 +498,13 @@ const styles = {
         alignItems: "center"
 
     },
+    image:{
```
Comment (isuruAb) [inline src/components/SkillEditor/SkillEditor.js]: what is this part ? could you please explain me?

Comment (isuruAb) [inline src/components/SkillEditor/SkillEditor.js]: where did you use this ?

File: src/components/SkillEditor/SkillEditor.js [hunk 2]
```diff
@@ -411,7 +399,7 @@ export default class Container extends React.Component {
                                     value={this.state.expertValue}
                                     onChange={this.handleExpertChange}
                                 />
-                                {this.state.showImage && <img alt="preview" id="target" style={{ width: 60, height: 60, borderRadius: "50%", marginRight: 20, border: 0 }} src={this.state.image} />}
+                                {this.state.showImage && <img alt="preview" id="target" style={styles.image} src={this.state.image} />}
```
Comment (rishiraj824) [inline src/components/SkillEditor/SkillEditor.js]: @isuruAb It's used to remove the inline styling.

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by mariobehling** [comment]
Travis is failing.

**Comment by rishiraj824** [comment]
@mariobehling Please merge now. I have fixed the travis and conflicts.

**Comment by rishiraj824** [comment]
@mariobehling Please merge now. I have fixed the travis and conflicts.

**Pull request merged**